.gitignore

```diff
@@ -123,3 +123,6 @@ ansible_collections/arista/avd/molecule/*/intended
 ansible_collections/arista/avd/molecule/*/documentation
 ansible_collections/arista/avd/molecule/*/config_backup
 !ansible_collections/arista/avd/molecule/eos-cli-config-gen/*
+
+#  playground for tests during development
+playground
```

Comment:
Maybe call this "dev" instead?

Reply:
+1 call this "development"

Reply:
We have development already with number of files. Playground is completely gitignored and I keep my tests and temp files there. It's not a must to have it as part of AVD.